lib/puppet-strings/markdown/templates/table_of_contents.erb

```diff
@@ -6,20 +6,20 @@ _Public <%= group_name %>_
 
 <% group.each do |item| -%>
 <% unless item[:private] -%>
-* [`<%= item[:name] %>`](#<%= item[:link] %>): <%= item[:desc] %>
+* [`<%= item[:name] %>`](#<%= item[:link] %>)<% unless item[:desc].empty? %>: <%= item[:desc] %><% end %>
```

Comment:
Is `item[:desc]` guaranteed to not be `nil`?

Reply:
> Is `item[:desc]` guaranteed to not be `nil`?

I wouldn't count on it...

```suggestion
* [`<%= item[:name] %>`](#<%= item[:link] %>)<% unless item[:desc].nil? || item[:desc].empty? %>: <%= item[:desc] %><% end %>
```